tests/data/basic4.tsv

```diff
@@ -1,7 +1,7 @@
 #license: "https://creativecommons.org/publicdomain/zero/1.0/"
 #mapping_set_id: https://w3id.org/sssom/mapping/tests/data/basic4.tsv
 #mapping_tool: "https://github.com/cmungall/rdf_matcher"
-#creator_id2: "cjm"
+#creator_id2: "orcid:1234"

```

Comment:
creator_id2 should not be allowed in header - right now validation of non defined properties is pretty lax.

Reply:
(this does not have to solved in this PR!)